### PR TITLE
Cortx-32105: Codacy pep257

### PR DESCRIPTION
**Pull request by rahul-tripathi-git**
# Problem Statement
- PEP 257 issues

# Design

(D104): Missing docstring in public package.
(D105): Missing docstring in magic method.
(D107): Missing docstring in init.
(D203): 1 blank line required before class docstring (found 0).
(D211): No blank lines allowed before class docstring (found 1).
(D212): Multi-line docstring summary should start at the first line.
(D213): Multi-line docstring summary should start at the second line.

D211 and D203 are conflicting, All D203 should be ignored, and all D211 are wrongly-reported.
D212 and D213 are conflicting. All D212 should be ignored and all D213 are wrongly-reported.
Also following error codes will be ignored:
(D203), (D212), (D107), (D104), (D105).

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing
- Are test cases updated along with code changes due to Enhancements/Bugs [Y/N]:N/A
- Confirm that new test cases are added to regression and sanity plan files and relevant feature plan files [Y/N]: N/A
- Confirm that Test Cases are added for new features added [Y/N]: N/A
- Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: N/A
- Confirm Testing was performed with installed RPM/K8s deployment [Y/N]:  N/A

# Review Checklist 
####  Before posting the PR please ensure:
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: N
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]: N/A
- [ ] New package/s added to setup.py?:  N/A
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [x] All the tests performed should be mentioned before Resolving a JIRA. :  N/A
- [x] Verification needs to be done before marked as Closed/Verified.

# KVstore/Confstore feature (changes/fixes) checklist
#### Confirm changes are made for:  N/A
- [ ] ConfStore
- [ ] KVStore
- [ ] ConfCli
- [ ] Test cases added for all above 3
- [ ] changes done in all the KVpayloads (ConsulKvPayload, IniKvPayload, KvPayload)

# Documentation
- [ ] Changes done to WIKI / Confluence page:  N/A

### CODE REVIEW

File: py-utils/src/utils/conf_store/conf_cli.py [hunk 1]
```diff
@@ -332,7 +342,7 @@ class AddNumKeysCmd:
     @staticmethod
     def add_args(sub_parser) -> None:
         s_parser = sub_parser.add_parser('addnumkeys', help=
-            "Adds num keys for the list items" \
+            "Adds num keys for the list items"\
```
Comment (rohit-k-dwivedi) [inline py-utils/src/utils/conf_store/conf_cli.py:345]: space is required here, else it will display `itemsExample`

**Review by rohit-k-dwivedi — APPROVED** [review]

**Comment by rahul-tripathi-git** [comment]
**Please note:**
> (D104): Missing docstring in public package (D104)
(D105): Missing docstring in magic method
(D107): Missing docstring in init
(D212): Multi-line docstring summary should start at the first line
(D213): Multi-line docstring summary should start at the second line

- D211 and D203 are conflicting, All D211 should be ignored, and all D203 are miss-reported.
- D212 and D213 are conflicting. All D212 should be ignored and all D213 are miss-reported.
- Also following error codes will be ignored:
- (D107), (D104), (D105).

**Review by cdeshmukh — APPROVED** [review]

**Comment by cdeshmukh** [comment]
1N deployment failed https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Cortx-Deployment/job/Generic/job/py-utils/474/

**Comment by rohit-k-dwivedi** [comment]
> 1N deployment failed https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Cortx-Deployment/job/Generic/job/py-utils/474/

deployment was successful, altough one of the io test failed

```
[2022-06-24T12:19:39.664Z] 
  [2022-06-24T12:19:39.664Z] --------------- Remove 'test-bucket' bucket ---------------
  [2022-06-24T12:19:39.664Z] 
  [2022-06-24T12:19:40.599Z] remove_bucket failed: s3://test-bucket An error occurred (BucketNotEmpty) when calling the DeleteBucket operation: Unknown
  [Pipeline] }
```

full Log:
[deployment log](https://github.com/Seagate/cortx-utils/files/8980027/deployment.log)

**Comment by rahul-tripathi-git** [comment]
Thanks @rohit-k-dwivedi. That will be helpful!

**Comment by rahul-tripathi-git** [comment]
retest this please

**Comment by cdeshmukh** [comment]
@rahul-tripathi-git try deployment soon now as deployment issue in S3 was rectified

**Comment by cdeshmukh** [comment]
Contradicting Codacy warnings D203 & D211 seen for this PR.

**Comment by rahul-tripathi-git** [comment]
Created a story ticket on RE team for suppressing the mentioned ((D203), (D212), (D107), (D104), (D105).) error codes.
https://jts.seagate.com/browse/CORTX-32481

**Pull request merged**